Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -414,7 +414,8 @@ private async void SendRoundEndDiscordMessage()
 
                 content = Loc.GetString("discord-round-notifications-end-ping", ("roleId", DiscordRoundEndRole));
                 payload = new WebhookPayload { Content = content };
-                payload.AllowedMentions.AllowRoleMentions();
+                payload.AllowedMentions = new();
+                payload.AllowedMentions.Value.AllowRoleMentions();
 
```

Comment:
No role mentions are being used
```suggestion
                payload.AllowedMentions = new();

```

Reply:
This is for the round end pings not the votes, this one uses roles.

Reply:
It was only updated because AllowedMentions is now null so it needs to make the list first